includes/class-comment.php

```diff
@@ -340,6 +341,31 @@ public static function comment_class( $classes, $css_class, $comment_id ) {
 		return $classes;
 	}
 
+	/**
+	 * Filters the comment feed where clause to exclude ActivityPub comment types.
```

Comment:
```suggestion
	 * Makes the comment feed filterable by comment type.
	 *
	 * Also excludes ActivityPub comment types from the feed when no type is specified.
```
Not sure this is suggestion is the best way to convey that information, but it would be nice to reflect both aspects of what this function does.